ssl/quic/quic_channel_local.h

```diff
@@ -107,6 +107,13 @@ struct quic_channel_st {
      */
```

Comment:
Isn't the comment here out of date? It talks about the first Initial packet the client transmitted - but isn't this updated after a retry?

Reply:
I suppose thats debatable.  When a retry packet is received, the connection sends another Initial packet, which is really the first 'real' Initial packet of the connection, as the previous one is required to be discarded by the server, so you could argue that the init_dcid is still the init_dcid for the connection, as the prior dcid was forgotten